frontend/pages/organizations/[id]/about.vue

```diff
@@ -53,7 +53,7 @@
           :organization="organization"
         />
         <div class="h-full w-full">
-          <MediaImageCarousel :class="{ 'lg:hidden': textExpanded }" />
+          <ModalMediaImageCarousel :class="{ 'lg:hidden': textExpanded }" />
```

Comment:
We need to make sure to include this so that the image carousel disappears if there's enough text in the About section to have a `View full text` button and then it's clicked :)